purview/utils/general.py

```diff
@@ -292,3 +291,76 @@ def get_node_count(query):
         return results_dict
 
     return results[0][0]
+
+
+def get_all_stories(query, resources_to_expand=None):
```

Comment:
Let's try consolidating this into the view function directly since it's not being reused anywhere.

Reply:
The view function will then grow very large, do we still want to do it?